includes/definitions/huaweiups.yaml

```diff
@@ -6,9 +6,10 @@ icon: huawei
 rfc1628_compat: true
 over:
     - { graph: device_current, text: Current }
+    - { graph: device_voltage, text: Voltage }
+    - { graph: device_load, text: Load }
 discovery:
     -
-      sysDescr: Linux GSE200M
       snmpget:
```

Comment:
@PipoCanaja we try to reduce the number of snmpgets in os detection by having a qualifier before the snmpget.

Reply:
OK I'll add a list for sysDescr match. Cause Huawei does not put something constant in sysDescr for all their UPSes, instead they are quite clean in UPS-MIB.